src/components/dataset/details/VisualizeRelationshipsPanel.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Paper, Typography, Button } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { Launch } from '@material-ui/icons';
+
+const styles = (theme) => ({
+  root: {
+    height: '100%',
+  },
+  headerText: {
+    fontWeight: theme.typography.bold,
+    textTransform: 'uppercase',
+  },
+  flex: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    margin: `${theme.spacing(2)}px 0px`,
+  },
+  jadeLink: {
```

Comment:
I wonder, if we need to do this everywhere, should this be put into a `JadeLink` component?